framework/source/class/qx/ui/basic/Label.js

```diff
@@ -391,6 +406,9 @@ qx.Class.define("qx.ui.basic.Label",
       if (this.__webfontListenerId) {
         this.__fixEllipsis();
       }
+      if (rich && this.getBreakWithinWords()) {
+        styles.wordBreak = "break-all";
```

Comment:
is this supported on all browsers ?

Reply:
yes, IE6 onwards https://caniuse.com/#search=word-break